product_docs/docs/pgd/5/node_management/witness_nodes.mdx

```diff
@@ -2,34 +2,33 @@
 title: Witness nodes
 ---
 
-A witness node is a lightweight node which functions as a data node but which
-does not store or replicate data. It is used to allow a PGD cluster which uses
+A witness node is a lightweight node that functions as a data node but that
+doesn't store or replicate data. Use a witness node to allow a PGD cluster that uses
 Raft consensus to have an odd number of voting nodes and therefore be able to
 achieve a majority when making decisions.
 
-## Witness Nodes within PGD groups or regions
+## Witness nodes within PGD groups or regions
 
 One typical use of witness nodes is when a PGD group has two data nodes but
-resources are not available for the recommended three data nodes. In this case,
-a witness node can be added to the PGD group to provide a third voting node to
-local Raft decision making. These decisions are primarily about who will be
-electing a write leader for the proxies to use. With only two nodes, there could
-be no consensus over which data node could be write leader. With two data nodes
-and a witness, there would be two candidates (the data nodes) and three voters
-(the data nodes and the witness) normally, and when a data node was down, there
-would still be two voters able to select a write leader.
+resources aren't available for the recommended three data nodes. In this case,
+you can add a witness node to the PGD group to provide a third voting node to
+local Raft decision-making. These decisions are primarily about who will be
+electing a write leader for the proxies to use. With only two nodes, it's possible 
+to have no consensus over which data node is write leader. With two data nodes
+and a witness, there are two candidates (the data nodes) and three voters
+(the data nodes and the witness) normally. When a data node is down, then, there
+are still two voters that can select a write leader.
```

Comment:
I wasn't strongly opposed to the conditional tense, but I found it hard to read. Is this revision clear and accurate to you?

Reply:
This reads well to me when looking at the paragraph without seeing the source text. The only thing I would changes is to delete "normally". 

What happens when the situation is not "normal"?